generate/genhcl/genhcl.go

```diff
@@ -16,7 +16,7 @@
 package genhcl
 
 import (
-	stdmt "fmt"
```

Comment:
o.o I did this? hahaha sorry

Reply:
Didnt see it on the review either, brain autocompletes I suppose =P